### ISSUE TITLE
Volume slider in Chromecast remote is too close to bottom.

### ISSUE THREAD

**Issue by Moohan** [jellyfin/jellyfin-android#624]
I have gesture navigation on my phone (I think this is Android 11+?). One gesture is to swipe the bottom to switch apps.

When using the Jellyfin Chromecast remote in the Android app, the volume slider is very close to the bottom of the screen meaning it's quite easy to accidentally change apps instead of changing the volume.

I think this could be fixed by moving the slider and buttons (CC, shuffle, repeat) up a bit. Or by moving the slider above the buttons.

**Comment by WibblyGhost** [comment]
I also agree with this enhancement. As a suggestion from me I think that we should move all three icons up to the middle of the blank space and increase the size like around 200% _(maybe even 300%)_ as I cannot for the life of me click the subtitles button. Right now there is a huge amount of blank space down the bottom and especially on mobile devices all real-estate should be used. Sadly I have no coding experience in Android otherwise I would have a look at the issue.

Most of these icons need resizing and repositioning since all of them don't suit a touch interface at all and require a moues to use.

![Screenshot_20220310-194052](https://user-images.githubusercontent.com/47839859/157605485-b9405fa5-cc38-4e26-a0b6-1ca55d7b81c6.png)

**Comment by nielsvanvelzen** [comment]
These controls are part of jellyfin-web (and written in HTML/CSS/JS). Moving the issue.
